plugins/janus_nosip.c

```diff
@@ -1988,7 +1993,7 @@ static void *janus_nosip_relay_thread(void *data) {
 			!g_atomic_int_get(&session->destroyed) && !g_atomic_int_get(&session->hangingup)) {
 		if(session->media.updated) {
 			/* Apparently there was a session update */
-			if(have_server_ip && (inet_aton(session->media.remote_ip, &server_addr.sin_addr) == 0)) {
+			if(have_server_ip && (inet_aton(session->media.remote_ip, &server_addr.sin_addr) != 0)) {
```

Comment:
Aaah, good catch: I checked the SIP and SIPre plugins and they're already correct, so I guess it was either a copy/paste error at the time, or a failure to fix this one as well when we fixed those.